extensions/gdx-setup/src/com/badlogic/gdx/setup/ExternalExtensionsDialog.java

```diff
@@ -0,0 +1,244 @@
+        buttonPanel.add(buttonCancel, new GridBagConstraints(1, 0, 1, 1, 0, 0, CENTER, HORIZONTAL, new Insets(0, 0, 0, 0), 0, 0));
+        bottomPanel.add(buttonPanel, new GridBagConstraints(3, 0, 1, 1, 1, 1, SOUTHEAST, NONE, new Insets(0, 0, 0, 0), 0, 0));
+
+        contentPane.add(topPanel, new GridBagConstraints(0, 0, 1, 1, 1, 1, NORTH, BOTH, new Insets(0, 0, 0, 0), 0, 0));
+        contentPane.add(content, new GridBagConstraints(0, 1, 1, 1, 1, 1, NORTH, BOTH, new Insets(0, 0, 0, 0), 0, 0));
+        contentPane.add(bottomPanel, new GridBagConstraints(0, 2, 4, 1, 1, 1, SOUTH, HORIZONTAL, new Insets(0, 0, 0, 0), 0, 0));
+
+        warningNotice = new JLabel("List of third party extensions, not maintained by libGDX team");
+        warningNotice.setHorizontalAlignment(JLabel.CENTER);
+        topPanel.add(warningNotice, new GridBagConstraints(0, 0, 1, 1, 1, 0, NORTH, HORIZONTAL, new Insets(0, 0, 0, 0), 0, 0));
+
+        JSeparator separator = new JSeparator();
+        separator.setForeground(new Color(85, 85, 85));
+        separator.setBackground(new Color(85, 85, 85));
+
+        topPanel.add(separator, new GridBagConstraints(0, 1, 4, 1, 1, 1, NORTH, HORIZONTAL, new Insets(10, 0, 0, 0), 0, 0));
+
+        try {
+            initData();
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+
+    }
+
+    private void initData() throws ParserConfigurationException, IOException, SAXException {
+        DocumentBuilderFactory dbFactory = DocumentBuilderFactory.newInstance();
+        DocumentBuilder builder = dbFactory.newDocumentBuilder();
+        Document doc = builder.parse(ExternalExtensionsDialog.class.getResourceAsStream("/com/badlogic/gdx/setup/data/extensions.xml"));
+
+        doc.getDocumentElement().normalize();
+
+        NodeList nList = doc.getElementsByTagName("extension");
+
+        for(int i = 0; i < nList.getLength(); i++) {
+            Node nNode = nList.item(i);
+            if (nNode.getNodeType() == Node.ELEMENT_NODE) {
+
+                Element eElement = (Element) nNode;
+                String name = eElement.getElementsByTagName("name").item(0).getTextContent();
+                String description = eElement.getElementsByTagName("description").item(0).getTextContent();
+                String version = eElement.getElementsByTagName("version").item(0).getTextContent();
+
+                String rowText = name + " - " + description + " (v" + version + ")";
+
+                HashMap<String, String> dependencies = new HashMap<String, String>();
+
+                addToDependencyMapFromXML(dependencies, eElement, "core");
+                addToDependencyMapFromXML(dependencies, eElement, "desktop");
+                addToDependencyMapFromXML(dependencies, eElement, "android");
+                addToDependencyMapFromXML(dependencies, eElement, "ios");
+                addToDependencyMapFromXML(dependencies, eElement, "html5");
```

Comment:
In our Gradle scripts that are generated and in the rest of the setup, as well as LibGDX codebase we call the html project "html" rather than "html5".